test/contrib/models/test_efficient_vit.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+import pytest
+import torch
+
+from kornia.contrib.models.efficient_vit import EfficientViTBackbone, efficientvit_backbone_b0
+
+
+class TestEfficientViT:
+    @pytest.mark.parametrize("img_size,expected_resolution", [(224, 7), (256, 8)])
+    def test_smoke(self, device, dtype, img_size: int, expected_resolution: int):
+        model: EfficientViTBackbone = efficientvit_backbone_b0(device=device, dtype=dtype)
+        image = torch.randn(1, 3, img_size, img_size, device=device, dtype=dtype)
+
+        out = model(image)
+
+        assert "input" in out
+        assert out["input"].shape == image.shape
+
+        assert "stage_final" in out
+        assert out["stage_final"].shape == torch.Size([1, 128, expected_resolution, expected_resolution])
+
```

Comment:
some smoke test, to check if it just builds/create the others backbone will be  nice

Reply:
done